core/editor.js

```diff
@@ -1648,6 +1648,83 @@
 
 		return element;
 	};
+
+	/**
+	 * Delay editor creation if given element is detached from DOM.
+	 *
+	 * Requires `config.delayIfDetached` to be equal to `true`.
+	 *
+	 * If `config.delayIfDetached_callback` is declared as function, it will be invoked with a single argument:
```

Comment:
OK, so I decide to change docs instead of code logic. Actually, we don't verify other config options that much. We just trust that callback is a function.

Reply:
That's true, we usually trust that the provided value will be of valid type. So in this context we could mention it as a function I suppose. I will correct the docs here :+1: